src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -247,6 +240,33 @@ final case class TenantApiServiceImpl(
     }
   }
 
+  override def internalRevokeCertifiedAttribute(
+    tenantOrigin: String,
+    tenantExternalId: String,
+    attributeOrigin: String,
+    attributeExternalId: String
+  )(implicit contexts: Seq[(String, String)], toEntityMarshallerProblem: ToEntityMarshaller[Problem]): Route =
+    authorize(INTERNAL_ROLE) {
+      val operationLabel =
+        s"Revoking certified attribute ($attributeOrigin/$attributeExternalId) from tenant ($tenantOrigin/$tenantExternalId)"
+      logger.info(operationLabel)
+
+      val result: Future[Unit] = for {
+        result <- revokeCertifiedAttribute(
+          tenantOrigin = tenantOrigin,
+          tenantExternalId = tenantExternalId,
+          attributeOrigin = attributeOrigin,
+          attributeExternalId = attributeExternalId
+        )
+        (tenant, attribute) = result
+        _ <- agreementProcessService.computeAgreementsByAttribute(tenant.id, attribute.id)
```

Comment:
I assume that this decision comes from the m2m implementation.
This looks good, but I would like confirmation about this behavior.